### ISSUE TITLE
Hidden recipient support

### ISSUE THREAD

**Issue by ArchieT** [google/end-to-end#322]
I tried to decrypt a message encrypted by me using GnuPG v2 to my friend (visible recipient) and me (hidden). It said that "No keys found for message".

**Comment by sirdarckcat** [comment]
Could you please replicate with a dummy message? We support wildcard keyids so it should work

**Comment by koto** [comment]
I don't think it works now. EncryptedMessage.decrypt asks for a single key from the KeyRing based on KeyID, so we don't iterate over all keys. https://github.com/google/end-to-end/blob/3288abc94b3c7b9adc1258a427f779dd584ff50e/src/javascript/crypto/e2e/openpgp/contextimpl.js#L325

**Comment by ArchieT** [comment]
Just use gpg2 -aeR [your hidden recipient] [your file].
